### PR TITLE
🧹 Remove unused settings

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/settings_common.py [hunk 1]
```diff
@@ -89,8 +88,6 @@
 TIME_ZONE = "GMT"
 USER_TIME_ZONE = "Africa/Kigali"
 
-MODELTRANSLATION_TRANSLATION_REGISTRY = "translation"
```
Comment (rowanseymour) [inline temba/settings_common.py:92]: hasn't been used since we got rid of django-modeltranslation

File: temba/settings_common.py [hunk 2]
```diff
@@ -216,10 +208,6 @@
     "temba.middleware.TimezoneMiddleware",
 )
 
-# security middleware configuration
-SECURE_CONTENT_TYPE_NOSNIFF = True
-SECURE_BROWSER_XSS_FILTER = True
```
Comment (rowanseymour) [inline temba/settings_common.py:221]: these and other security settings are in `security_settings.py`

File: temba/settings_common.py [hunk 3]
```diff
@@ -1178,12 +1163,7 @@
 MAILROOM_URL = None
 MAILROOM_AUTH_TOKEN = None
 
-# To allow manage fields to support up to 1000 fields
-DATA_UPLOAD_MAX_NUMBER_FIELDS = 4000
-
-# When reporting metrics we use the hostname of the physical machine, not the hostname of the service
-MACHINE_HOSTNAME = socket.gethostname().split(".")[0]
```
Comment (rowanseymour) [inline temba/settings_common.py:1185]: Only used by our Librato settings https://github.com/nyaruka/textit/pull/308

File: temba/settings_common.py [hunk 4]
```diff
@@ -120,11 +117,6 @@
 # calendars according to the current locale
 USE_L10N = True
 
-# URL prefix for admin static files -- CSS, JavaScript and images.
-# Make sure to use a trailing slash.
-# Examples: "http://foo.com/static/admin/", "/static/admin/".
-ADMIN_MEDIA_PREFIX = "/static/admin/"
```
Comment (rowanseymour) [inline temba/settings_common.py:126]: we don't use django-admin

**Review by norkans7 — APPROVED** [review]

**Pull request merged**